util/difgen/gen_dif_listing.py

```diff
@@ -70,7 +71,7 @@ def gen_listing_html(combined_xml, dif_header, dif_listings_html):
 # Generate HTML link for single function, using info returned from
 # get_difref_info
 def gen_difref_html(function_info, difref_html):
-    difref_html.write('<a href="{full_url}" title="{description}">'.format(**function_info))
+    difref_html.write('<a href="{full_url}">'.format(**function_info))
```

Comment:
The description here is remove as it can contain HTML, which we can't have in an attribute. Instead of inventing another stripping routine I just removed it, I wouldn't expect anybody to be too worried about it.